controllers/app.controller.js

```diff
@@ -1,10 +1,13 @@
 const { readTopics } = require("../models/app.model");
+const endpointData = require("./endpoints.json");
 
 function getTopics(req, res, next) {
-  readTopics()
-    .then((topics) => {
-      res.status(200).send(topics);
-    })
+  readTopics().then((topics) => {
+    res.status(200).send(topics);
```

Comment:
Any data sent from a controller needs to be sent back on a meaningful key ie. .send({topics})